### ISSUE TITLE
The Joplin window is positioned off screen when changing from dual displays to a single display

### ISSUE THREAD

**Issue by kenclark75** [laurent22/joplin#6882]
I am using Joplin v2.8.8 on a 14" laptop PC running Windows 11.
When at home, I extend the display onto a 24" external monitor and do most of my work on this screen. When travelling, only the laptop screen is available.

If I close Joplin whilst it is running on the external display, it "remembers" this location.
If I then run the application with no external display connected, the Joplin window is not displayed on the laptop.

The situation can be recovered by re-connecting an external monitor and dragging the Joplin window back onto the laptop screen. If no monitor is available, however, my only remedy is to uninstall and re-install the application.

The obvious solution is to ensure that I always move the window back onto the laptop screen before closing the application. Unfortunately, I know that I am fallible.
Is there a configuration file which I can manually edit to change the window location?

**Comment by Daeraxa** [comment]
Press `windows + shift + left/right`? That is a generic Windows shortcut to move a window between monitors.

**Comment by kenclark75** [comment]
Thanks for the prompt response.
I regularly use that key combination to switch between two displays.  However, my problem occurs when only my laptop screen is available. In that situation, it has no effect.

The following may help to clarify the issue:

- Using my laptop at home with an external monitor connected.
- Open Joplin and move its window onto the external monitor.
- Close Joplin then shut down the laptop and unplug the monitor.
- Take the laptop to a different location where no external monitor is available.
- Boot the computer and open Joplin. The Joplin icon appears in the taskbar but there is no window on the screen. If I hover the cursor over the icon, it shows a thumbnail of the window. I have tried using win+shift+left/right but it has no effect.
- If I am able to borrow and connect an external monitor, the Joplin window immediately appears on this monitor.  I can then move the window to the laptop screen, either by dragging or using the key combination. I then close Joplin, shut down the computer and disconnect the monitor.
- If I now reboot the computer, Joplin will run normally on the laptop screen.

Now that I am aware of the issue, I can avoid it by ensuring that I always move the Joplin window back to the laptop screen before closing it.  However, I only travel occasionally so it would be easy to become careless about this procedure.
It would be helpful if I knew how to edit a configuration file to reset the screen position.
-

**Comment by kenclark75** [comment]
The emphasis on the last three sentences was accidental - must have hit a wrong key!

**Comment by Daeraxa** [comment]
Strange, I do a similar thing with my work laptop when removing it from the dock with external monitors but never had an issue with that key combo bringing the window back even when it is "on" a non-existent screen.

I'm assuming this is controlled in the Windows registry rather than in Joplin, I don't think it is something stored in the regular config files at least.

There are other things I know you can try involving shift+right clicking the application icon in the task bar and selecting "move" which will then allow you to shift the window with the cursor keys.

**Comment by kenclark75** [comment]
Thanks for your further feedback and for the solution provided.

I do not think that this is a general Windows issue. I ran a test where I opened five other random applications in addition to Joplin. None of the other applications behaved in this way.

Fortunately, your tip about using "move" from the application icon works perfectly and completely solves the problem.
I was not previous aware that this menu was accessible from the icon - I am sure that I shall find other uses for this feature.

FYI, I only recently began using Joplin. I updated my laptop to Windows 11 22H2 on Tuesday evening so the update was in place when I first encountered the issue.  This may not be relevant, however, since I do not believe that similar circumstances ever occurred whilst I was using 21H2.
